assemble/bin/accumulo-cluster

```diff
@@ -100,15 +100,45 @@ function get_ip() {
   echo "$ip_addr"
 }
 
+function set_last_instance_id() {
+  if [[ "$service" == "tserver" ]]; then
+    last_instance_id=${NUM_TSERVERS:-1}
+  else
+    last_instance_id=1
+  fi
+}
+
+function set_service_instance_if_needed() {
+  if [[ "$service" == "tserver" && ${NUM_TSERVERS:-1} > 1 ]]; then
```

Comment:
Should use numeric greater-than comparison here, instead of ASCII lexical comparison:
```suggestion
  if [[ "$service" == "tserver" && ${NUM_TSERVERS:-1} -gt 1 ]]; then
```

Reply:
Thanks. I had run shellcheck initially but forgot to run it before pushing this round of updates. Indeed that caught this issue as well. I'll make sure I fix it.

Reply:
I saw you already fixed the "unnecesarry $ in math expression" issue. shellcheck is awesome, isn't it? :smiley_cat: